ui/lib/core/addon/components/replication-summary-card.js

```diff
@@ -0,0 +1,52 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import layout from '../templates/components/replication-summary-card';
+
+/**
+ * @module ReplicationSummaryCard
+ * ReplicationSummaryCard components
```

Comment:
looks like this needs the full description 😉

Reply:
Good catch.  This was missing on both the ReplicationSecondaryCard and ReplicationPrimaryCard components as well, so I went ahead and added them.